plugins/status.js

```diff
@@ -14,7 +51,29 @@ const statusPlugin = {
         server.route({
             method: 'GET',
             path: '/status',
-            handler: (_, h) => h.response('OK').code(200),
+            handler: async (request, h) => {
+                const { exhaustive } = request.query;
```

Comment:
Wondering if there's a different term we can use vs `exhaustive`. Seems easy to misspell.
No strong preference though

Reply:
extended ?

Reply:
I guess either `complete` for ease of use or just pick the word that best describes the intent (which could be `exhaustive` or `extensive` or whatever)

Reply:
let's stick with `exhaustive` for now